pkg/fieldpath/fieldpath.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
 )
 
+// formatMap formats map[string]string to a string.
+func formatMap(m map[string]string) string {
+	var l string
+	for key, value := range m {
+		l += key + "=" + value + "\n"
```

Comment:
Pedantic: This is ambiguous syntax.  If a value string has a newline in it (which is legit) the parsing of this file will fail.  If you want to use newline as a delimiter you have to escape it in values.  And have tests.